### PR TITLE
[6]Product Page using HTML

### PR DESCRIPTION
**Pull request by ChabuChabu**
## Task: `Created products page`
- `Created product page with two images and its description
kwathu-product-page

### CODE REVIEW

**Comment by adaugochi** [comment]
Well done @ChabuChabu . Please rename your folder name from `html-product-page` to just `html`

File: submissions/chabuChabu/html-products-page/product.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" >
+        <title>Products-page</title>
+    </head>
+    <body bgcolor="#e6faff">
+
+
+        <h1 align ="center">Kwathu.!!!</h1><hr>
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-products-page/product.html:10]: Remove the style here

File: submissions/chabuChabu/html-products-page/product.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+         <div align="center">
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-products-page/product.html:12]: Same as above

File: submissions/chabuChabu/html-products-page/product.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+         	<h3>SAMSUNG  2020 49"Q85T QLED 4K HDR</h3>
+         </div>
+
+
+                
+         <table border="0" align="center" cel>
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-products-page/product.html:18]: Same as above

File: submissions/chabuChabu/html-products-page/product.html [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            
+             <tr align="center">
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-products-page/product.html:20]: Same as above

File: submissions/chabuChabu/html-products-page/product.html [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+                 
+
+             </tr>
+         </table>
+
+         <h3 align="center"><em>Price</em> :<strong>ZMW 8888</strong><br></>
+
+         <h3 align="center"><strong>Prouct description</strong></h3>
+
+         <table border="0" align="center" cellspacing ="0" cellpadding ="10">
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-products-page/product.html:42]: Same here

File: submissions/chabuChabu/html-products-page/product.html [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+            
+             <tr align="left">
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-products-page/product.html:44]: Same as above

File: submissions/chabuChabu/html-products-page/product.html [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+                 <td>
+                     <img src="product-02.jpg" width="300" height="230"><br>
+                  
+                </td>
+                 
+
+             </tr>
+         </table>
+
+         <h3 align="center"><em>Price</em> :<strong>ZMW 8888</strong><br></>
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-products-page/product.html:38]: Same here and the next `<h3>`

File: submissions/chabuChabu/html-products-page/product.html [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+
+                
+         <table border="0" align="center" cel>
+            
+             <tr align="center">
+                
+                
+                <td>
+                    <img src="product-01.jpg" width="300" height="230"><br>
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-products-page/product.html:24]: Remove the `width` and `height` from both images.

File: submissions/chabuChabu/html-products-page/product.html [hunk 9]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-products-page/product.html:1]: Well done! Remove the extra one or 2 extra lines in this file.

**Pull request closed without merging**